typescript/sdk/src/core/HyperlaneCore.ts

```diff
@@ -79,9 +79,10 @@ export class HyperlaneCore<
     const contracts = this.getContracts(chain);
     return {
       mailbox: contracts.mailbox.address,
-      // TODO allow this to be more easily changed
+      // TODO allow these to be more easily changed
       interchainGasPaymaster:
         contracts.defaultIsmInterchainGasPaymaster.address,
+      interchainSecurityModule: contracts.multisigIsm.address,
```

Comment:
Should we actually do this? Don't feel strongly would have biased to keep as is, i.e. you use the default via the mailbox, not by explicitly setting it

Reply:
For middlewares that we deploy we should definitely leave as deferring to the mailbox, as that's presumably the same trust model and means fewer things to configure if we ever change